app/routes.js

```diff
@@ -8,15 +8,39 @@ const csv = require('fast-csv');
 const data = require(path.join(__dirname, '/data', '/session-data-defaults.js'));
 const staff = data.staff.flat();
 
-router.get('/employee/:page', function (req, res) {
+/*router.get('/employee/:page', function (req, res) {
   const version = req.query.version;
   const figma = req.query.figma;
+  const email = req.query.email ?? 'name@email.com';
   let path = `employee/${req.params.page}`;
   if (req.params.page === 'start' && version === 'a') {
     path += '-version-a';
   }
-  res.render(path, { version, figma })
-})
+  if (req.params.page === 'how-to-contact-you' && req.session.data.version === 'b') {
+    path += '-version-b';
+  }
+  res.render(path, { version, figma, email })
+});*/
```

Comment:
Is it worth just removing this rather than commenting out?  We'll always have it in the git history if we need it again?